src/domain/individualApplication/IndividualApplicationPage.tsx

```diff
@@ -22,7 +22,7 @@ import OfferCard, { OfferCardProps } from './offerCard/OfferCard';
 
 export enum CUSTOMER_GROUP {
   PRIVATE = 'PRIVATE',
-  COMPANY = 'COMPANY',
+  ORGANIZATION = 'ORGANIZATION',
```

Comment:
if it is `organization`, please check what type and use that as `CUSTOMER_GROUP` 🙂